--- StrongReference.onone.sil
+++ StrongReference.leaky.sil
@@ -108,7 +108,6 @@
   release_value %58                               // id: %61
   release_value %56                               // id: %62
   release_value %54                               // id: %63
-  strong_release %32                              // id: %64
   %65 = integer_literal $Builtin.Word, 1          // user: %67
   // function_ref _allocateUninitializedArray<A>(_:)
   %66 = function_ref @$ss27_allocateUninitializedArrayySayxG_BptBwlF : $@convention(thin) <τ_0_0> (Builtin.Word) -> (@owned Array<τ_0_0>, Builtin.RawPointer) // user: %67
